code-challenges.test.js

```diff
@@ -23,10 +31,33 @@ const secretCodeWord3 = "Eccentric"
 
 // b) Create the function that makes the test pass.
 
+// Process: created a function that takes in a string as an input, and replaces specified letters with numbers
+// Encrypter: we tell the function that our string is gonna have the specifies letter replaces by using the .replace method and we specify each letter and what is the replacement we want
+// Display: we run our function alongside our code word to display expected outcome
+
+const encrypter = (str) => {
+    return(
+    str = str.replace(/a/gi, 4),
+    str = str.replace(/e/gi, 3),
+    str = str.replace(/i/gi, 2),
+    str = str.replace(/o/gi, 0)
+    )
+}
+console.log(encrypter(secretCodeWord1))
+console.log(encrypter(secretCodeWord2))
+console.log(encrypter(secretCodeWord3))
+
 // --------------------2) Create a function that takes in an array of words and a single letter and returns an array of all the words containing that particular letter.
 
 // a) Create a test with expects statement using the variable provided.
 
+// describe('takes in an arr of words & specified letter and returns arr containing words w/ specified letter', () => {
+//     it("returns arr with only words containing specified letter", () => {
+//         expect(fruitFilter(filterLetterA).toEqual(["Mango", "Apricot", "Peach"]))
```

Comment:
The function should take 2 parameters, the array and the letter. Here, you are only giving it 1